src/content/1.7/development/behind-the-scenes/orders/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Orders
+
+weight: 42
+
+chapter: true
+---
+
+# Orders related code specifics
+
+Explains some "not so obvious" technical parts related to orders domain.
+
+### Context state manager
```

Comment:
if all bellow items goes to order create page, then maybe we could find a place for context state manager somewhere more "general". and remove this "behind-the-scenes/orders" at all?